### PR TITLE
Feature Wikis

### PR DESCRIPTION
**Pull request by loicteixeira**
Public wikis

### User Stories
- [X] As a user with a standard account, I want to create, read, update, and delete public wikis

### Test Required
- [X] Are you able to create new public wikis?
- [X] Are you able to update public wikis?
- [X] Are you able to delete public wikis?

Note: As agreed, I did not write specs and did the testing by hand.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+users = []
+%w(R2-D2 C-3PO Bender Robocop GLaDOS Wall-E Marvin).each do |robot_name|
```
Comment (eliotsykes) [inline db/seeds.rb]: Contribution to faker?

**Comment by eliotsykes** [comment]
Excellent work. 

Do the wikis need protecting from changes from unauthorized users? I'm not certain what requirements you've decided on around who can edit what.

File: db/seeds.rb (hunk 1 above)
Comment (loicteixeira) [inline db/seeds.rb]: Ahah that would be cool. Unfortunately, looking at the repo, there are several month old pull-request (adding Game Of Thrones characters names or maybe more useful adding countries or name initials) so I'm not very confident in my pull-request getting merged. That might still be a good exercise though.

**Comment by loicteixeira** [comment]
The user story was "As a user with a standard account, I want to create, read, update, and delete public wikis". Although I restricted the deletion to admins in the upcoming user roles checkpoint.

**Pull request merged**